### PR TITLE
Add some basic docker documentation on how to ssh into the container #584

### LINKED ISSUE #584: Add some basic docker documentation on how to ssh into the container

### PR DESCRIPTION
**Pull request by amczarny**
fixes #584. Instead of adding documenation I assume it is simpler to have an extra flag --debug-docker which allows you to start docker in interactive mode pluse extra info how to run build script

```
DEBUG DOCKER BUILD
To build jdk run
/openjdk/sbin/build.sh
```

DCO 1.1 Signed-off-by: Andrzej Czarny

### CODE REVIEW

File: sbin/build.sh [hunk 1]
```diff
@@ -521,25 +521,43 @@ showCompletionMessage()
   echo "All done!"
 }
 
+execute()
```
[the remark this answers is not in the document]
Comment (amczarny) [reply sbin/build.sh]: How about reverting all changes in sbin/build.sh.
All we need is to have access to docker with bash
We cen modify our entrypoint in docker-build.sh
```
local dockerMode=(-l "st")
local dockerEntrypoint=(--entrypoint /openjdk/sbin/build.sh "${BUILD_CONFIG[CONTAINER_NAME]}")
  if [[ "${BUILD_CONFIG[DEBUG_DOCKER]}" == "true" ]] ; then
     dockerMode=(-t -i -l "st")
     dockerEntrypoint=(--entrypoint "/bin/sh" "${BUILD_CONFIG[CONTAINER_NAME]}" -c "echo 'DEBUG DOCKER BUILD\nTo exeucte build run\n/openjdk/sbin/build.sh';/bin/bash")
  fi

  # shellcheck disable=SC2140
  # Pass in the last important variables into the Docker container and call
  # the /openjdk/sbin/build.sh script inside
  ${BUILD_CONFIG[DOCKER]} run \
       "${dockerMode[@]}" \
       --cpuset-cpus="${cpuSet}" \
       -v "${BUILD_CONFIG[DOCKER_SOURCE_VOLUME_NAME]}:/openjdk/build" \
       -v "${hostDir}/workspace/target":"/${BUILD_CONFIG[WORKSPACE_DIR]}/${BUILD_CONFIG[TARGET_DIR]}" \
       "${gitSshAccess[@]}" \
       -e DEBUG_DOCKER_FLAG="${BUILD_CONFIG[DEBUG_DOCKER]}" \
       -e BUILD_VARIANT="${BUILD_CONFIG[BUILD_VARIANT]}" \
       "${dockerEntrypoint[@]}"
``` 
as result we will get
```
DEBUG DOCKER BUILD
To exeucte build run
/openjdk/sbin/build.sh
build@32461b04ef08:/openjdk/build$ 
```

File: docker-build.sh [hunk 2]
```diff
@@ -127,15 +127,22 @@ buildOpenJDKViaDocker()
   if [[ "${BUILD_CONFIG[USE_SSH]}" == "true" ]] ; then
      gitSshAccess=(-v "${HOME}/.ssh:/home/build/.ssh" -v "${SSH_AUTH_SOCK}:/build-ssh-agent" -e "SSH_AUTH_SOCK=/build-ssh-agent")
   fi
-  
+ 
+  local dockerMode=(-l "st")
```
[the remark this answers is not in the document]
Comment (amczarny) [reply docker-build.sh]: to be remove by #648

File: sbin/build.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (karianna) [reply sbin/build.sh]: Yes I prefer this idea - nice workaround!

[the remark this answers is not in the document]
Comment (amczarny) [reply sbin/build.sh]: done

**Pull request merged**